addons/mail/static/src/models/composer/composer.js

```diff
@@ -269,6 +527,13 @@ function factory(dependencies) {
         thread: one2one('mail.thread', {
             inverse: 'composer',
         }),
+        _allSuggestedPartners: many2many('mail.partner', {
```

Comment:
What was the motivation for prefixing this with `_`? Private field? If so, don't. We didn't put a "visibility" policy on fields yet, and I would prefer not introducing one so close to merging.